tests/DependencyInjection/Compiler/ObjectAclManipulatorCompilerPassTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\DependencyInjection\Compiler;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\AdminBundle\Command\GenerateObjectAclCommand;
+use Sonata\AdminBundle\DependencyInjection\Compiler\ObjectAclManipulatorCompilerPass;
+use Sonata\AdminBundle\Util\ObjectAclManipulator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @author Olivier Rey
+ */
+class ObjectAclManipulatorCompilerPassTest extends TestCase
+{
+    public function testAvailableManagerFromServiceDefinitionClass(): void
+    {
+        $containerBuilder = $this->getContainerWithServiceClass();
+
+        $objectAclManipulatorCompilerPass = new ObjectAclManipulatorCompilerPass();
+
+        $objectAclManipulatorCompilerPass->process($containerBuilder);
+
+        $availableManagers = $containerBuilder->getDefinition(GenerateObjectAclCommand::class)->getArgument(1);
+
+        $this->assertArrayHasKey('sonata.admin.manipulator.acl.object.orm', $availableManagers);
+    }
+
+    public function testAvailableManagerFromServiceDefinitionWithParameterClass(): void
+    {
+        $containerBuilder = $this->getContainerWithParameterAsServiceClass();
+
+        $objectAclManipulatorCompilerPass = new ObjectAclManipulatorCompilerPass();
+
+        $objectAclManipulatorCompilerPass->process($containerBuilder);
+
+        $availableManagers = $containerBuilder->getDefinition(GenerateObjectAclCommand::class)->getArgument(1);
+
+        $this->assertArrayHasKey('sonata.admin.manipulator.acl.object.orm', $availableManagers);
+    }
```

Comment:
Are you familiar with dataProvider ?

Since you're making the same assertion in the test you can write
```
    /**
    * @dataProvider containerProvider
    */
    public function testAvailableManager($containerBuilder): void
    {
        $objectAclManipulatorCompilerPass = new ObjectAclManipulatorCompilerPass();

        $objectAclManipulatorCompilerPass->process($containerBuilder);

        $availableManagers = $containerBuilder->getDefinition(GenerateObjectAclCommand::class)->getArgument(1);

        $this->assertArrayHasKey('sonata.admin.manipulator.acl.object.orm', $availableManagers);
    }

    public function containerDataProvider(): array
    {
        return [
            [$this->getContainerWithServiceClass()],
            [$this->getContainerWithParameterAsServiceClass()],
        ]
    }
```